### PR TITLE
Bw/xblock view

### PR DESCRIPTION
**Pull request by muselesscreator**
**TL;DR -** [ A short summary of what this PR does and why ]
Updates the MFE view to display the xblock view of the mfe, with appropriate routing (minus env-friendly url).

JIRA: [JIRA-XXXX](https://openedx.atlassian.net/browse/JIRA-XXXX)

**What changed?**
Annoying logic to make the iframe show up at reasonable size.

image

- [ More in depth breakdown of changes ]
- [ Peripheral things that got changed ]
- [ etc... ]

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/openedx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [openassessment/\_\_init\_\_.py](https://github.com/openedx/edx-ora2/blob/master/openassessment/__init__.py#L4) and [package.json](https://github.com/openedx/edx-ora2/blob/master/package.json#L3)

**Testing Instructions**

[ How should a reviewer test this PR? ]

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Review by jansenk — APPROVED** [review]

**Review by leangseu-edx — APPROVED** [review]

**Comment by jansenk** [comment]
I approved, but generally the understanding is that approval is for the real changes and that you'll fix any test failures and quality issues rather than merging through. I get that we're under big time pressure, but at the very least broadcast the fact that you're introducing test failures and write up a ticket for fixing them later on.
